### PR TITLE
[release/v2.25] prepare v2.25.2

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
v2.25.2, this time even with a changelog.

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/assets/config/changelog.json
Comment (Waseem826) [inline modules/web/src/assets/config/changelog.json:1]: I think we should also include changelog from the dashboard PRs, especially the chromium bugfix for mat-select.

Comment (xrstf) [reply modules/web/src/assets/config/changelog.json:1]: Yeah yeah yea hyeah hyhheahahj I just forgot them, it's monday :D

**Review by Waseem826 — APPROVED** [review]
/approve

File: modules/web/src/assets/config/changelog.json
Comment (Waseem826) [reply modules/web/src/assets/config/changelog.json:1]: Perfect! 😅

**Pull request merged**